python-main.js

```diff
@@ -661,7 +614,7 @@ function web_editor(config) {
             'instructions': config.translate.code_snippets.instructions,
             'trigger_heading': config.translate.code_snippets.trigger_heading,
             'description_heading': config.translate.code_snippets.description_heading,
-            'snippets': EDITOR.getSnippets(),
+            'snippets': snippetManager.snippetMap.python_microbit,
```

Comment:
It's better to keep `EDITOR.getSnippets()` here (and remove the `snippetManager` line above) , as otherwise we are leaking ACE api calls outside the `pythonEditor` object.

Reply:
Changes made